extensions/src/main/java/org/pentaho/platform/plugin/services/importexport/CommandLineProcessor.java

```diff
@@ -703,9 +721,9 @@ private void logResponseMessage( String logFile, String path, ClientResponse res
       message.append( Messages.getInstance().getString( "CommandLineProcessor.INFO_REST_RESPONSE_RECEIVED",
         response.getEntity( String.class ) ) );
     }
-    System.out.println( message.toString() );
-    if ( logFile != null && !"".equals( logFile ) ) {
-      writeFile( message.toString(), logFile );
+    System.out.println( message );
+    if ( logFile != null && !logFile.isEmpty() ) {
```

Comment:
Why not StringUtils.isEmpty( logFile )?

Reply:
Deprecate StringUtils.isEmpty(Object) and replace remaining usage (e.g. with ObjectUtils.isEmpty)
https://github.com/spring-projects/spring-framework/issues/25945

Reply:
Done.